notification/endpoint/slack.go

```diff
@@ -45,10 +45,10 @@ func (s Slack) Valid() error {
 	if err := s.Base.valid(); err != nil {
 		return err
 	}
-	if s.URL == "" && s.Token.Key == "" {
+	if s.URL == "" {
```

Comment:
token is not required for a url, if URL is "" it shoudl be an eror for slack, token shoudn't matter